src/main/resources/image-api.env

```diff
@@ -10,8 +11,9 @@ META_PORT
 META_INITIAL_CONNECTIONS
 META_MAX_CONNECTIONS
 CONTACT_EMAIL
-SEARCH_ENGINE_ENV_TCP_PORT
-SEARCH_ENGINE_ENV_CLUSTER_NAME
+SEARCH_SERVER
+SEARCH_SIGN_REQUESTS
```

Comment:
Er forsåvidt "to-the-point" navngiving. Men tok meg langt inn i koden for å skjønne at dette har med kjøring lokalt eller på AWS å gjøre. Kunne vi endret navnet noe? Føler egentlig det hadde vært bedre med `RUN_IN_ENV_WITH_SIGNED_REQUESTS` el.n.sånt...